src/Presenters/FaqPresenter.php

```diff
@@ -9,18 +9,18 @@ class FaqPresenter
     /**
      * Get all categories of FAQ
      *
-     * @return object
+     * @return Collection|Category[]
      */
     public function getAllCategories()
     {
-        return Category::paginate(20);
+        return Category::paginate(Category::DEFAULT_PAGINATE);
     }
 
     /**
      * Create or update category of FAQ
      *
-     * @param array $data
-     * @param integer $id
+     * @param object $data
```

Comment:
This is still not fixed. We should not pass Request to other methods than Controller's.